### PR TITLE
Checked-in Xcode project followups

### PR DESCRIPTION
**Pull request by jfirebaugh**
cc @1ec5

### CODE REVIEW

File: platform/ios/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ios.xcodeproj/xcuserdata
+ios.xcworkspace/xcuserdata
```
Comment (1ec5) [inline platform/ios/.gitignore]: No `xcuserdata` should ever be committed in my opinion. The shared settings live alongside these directories, in `xcshareddata`.

Comment (jfirebaugh) [inline platform/ios/.gitignore]: Are you suggesting broadening the ignore rule to (unprefixed) `xcuserdata`?

Comment (1ec5) [inline platform/ios/.gitignore]: Yes.

**Comment by friedbunny** [comment]
screen shot 2016-04-18 at 3 45 18 pm

On every fresh repo clone, anyone who wants to run the iosapp demo will now have to create the scheme themselves (#4741). This is not difficult, but is certainly less than friendly.

**Comment by 1ec5** [comment]
Once #4728 lands, maybe we should instead discourage folks from setting this environment variable. As for ios-bench, since that application needs user intervention anyways to download the assets locally (#4740), we should have it wait until you set your access token inside the application, then start benchmarking.

**Pull request merged**